models/fmcib/utils/FMCIBRunner.py

```diff
@@ -0,0 +1,53 @@
+"""
+---------------------------------------------------------
+Author: Suraj Pia
+Email:  bspai@bwh.harvard.edu
+---------------------------------------------------------
+"""
+
+import json
+import torch
+from fmcib.models import fmcib_model 
+import SimpleITK as sitk
+from mhubio.core import Instance, InstanceData, IO, Module
+from fmcib.preprocessing import preprocess
+
+
+class FMCIBRunner(Module):
+    @IO.Instance()
+    @IO.Input('in_data', 'nrrd:mod=ct', the='Input NRRD file')
+    @IO.Input('in_mask', 'nrrd|json', the='Tumor mask for the input NRRD file')
+    @IO.Output('feature_json', 'features.json', "json", bundle='model', the='output JSON file')
```

Comment:
If we accept json as an input, I suggest we create a schema file and validate it properly. Then, why not have a schema for the output json, too? What do you think? Should be effortless to do but makes makes it crystal clear what to expect from the model output!